c7n/filters/metrics.py

```diff
@@ -174,8 +175,20 @@ def process_resource_set(self, resource_set):
                     EndTime=self.end,
                     Period=self.period,
                     Dimensions=dimensions)['Datapoints']
+
+            # In certain cases CloudWatch reports no data for a metric.
+            # If the policy specifies a fill value for missing data, add
+            # that here before testing for matches. Otherwise, skip
+            # matching entirely.
             if len(collected_metrics[key]) == 0:
-                continue
+                if 'missing-fillvalue' not in self.data:
```

Comment:
can we rename this to just `missing-value` and add to the doc string on its usage scenario.